docs/2.8.0/docs/daml-finance/instruments/structured-product.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+How To Use the Structured Product Extension Package
```

Comment:
In one of the team meetings during your holiday we discussed dropping the "Extension" nomenclature. We don't define anywhere in the docs what an extension package is, so I refactored the other pages to avoid using "extension".

```suggestion
How To Use the Structured Product Instrument Packages
```

I am also using the plural as we are effectively using two packages here (interface + implementation)

Reply:
Maybe we want to define Extension as the two packages here (interface + implementation). That would make it singular, which I think is less confusing (somebody might wonder what other packages we are referring to).

Reply:
let's discuss in the catchup?